samwise.js

```diff
@@ -12,170 +12,167 @@
     var autoplayDefault = 0;
```

Comment:
SCRIPT-I appreciate how well-structured and clean your code is. It's very easy to follow!